src/g_game.c

```diff
@@ -742,26 +743,13 @@ static void G_DoLoadLevel(void)
     headsecnode = NULL;
    }
 
+  critical = (demoplayback || demorecording || D_CheckNetConnect());
+  M_UpdateCriticalItems();
+
   // [crispy] pistol start
-  if (pistolstart)
+  if (CRITICAL(pistolstart))
```

Comment:
@fabiangreffrath How about adding pistolstart option to the QOL menu?

Reply:
Yes, but we need to separate these from the "harmless" qol features.